builder.json

```diff
@@ -38,8 +40,13 @@
     },
     "macos": {
       "+test_steps": [
-       "./integ_test.sh"
+        "./integ_test.sh"
+      ]
+    },
+    "linux":{
+      "!test_steps":[
+        "swift test"
       ]
     }
   }
-}
+}
```

Comment:
utterly trivial: https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files